src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -32,12 +38,17 @@
                     @await RenderSectionAsync("NavbarTop", required: false)
                 </div>
                 <ul class="nav navbar-nav">
-                    <li class="nav-item">
-                        <a class="nav-link"><span><i class="fa fa-user" aria-hidden="true"></i></span><span> @User.Identity.Name</span></a>
+                    <li class="nav-item dropdown">
```

Comment:
This should probably a shape that get's modified if the Gravatar module is enabled.

Reply:
This is not related to gravatari. IMHO is suited coz it injected and set the image and it's properties